### ISSUE TITLE
fallback not work

### ISSUE THREAD

**Issue by ipy** [getsentry/react-native-sentry#185]
In `raven.js` of the `raven-js` lib, `_drainPlugins` is in `install` method while the `globalOptions` assignment is in `config`. So `_globalOptions.transport`  won't be assigned to `globalOptions.transport`.
When native not available, it will use the default `transport` in `raven.js` instead of which in the react-native plugin. and because

1. hasCORS is false
2. Content-type header not set

the request will not send.

**Comment by HazAT** [comment]
Hey,
sorry, I think I don't fully understand the problem.

We set the transport here if the native plugin isn't available:
https://github.com/getsentry/react-native-sentry/blob/master/lib/raven-plugin.js#L81
and `raven-js` fetches the transport before every send:
https://github.com/getsentry/raven-js/blob/master/src/raven.js#L1583

So can you give me an example code/config which does not work?

**Comment by ipy** [comment]
```
import { Sentry } from 'react-native-sentry';
Sentry.config('xxx').install();
Sentry.captureMessage('test');
```

Sorry for my bad expression...  As you mentioned:
https://github.com/getsentry/react-native-sentry/blob/master/lib/raven-plugin.js#L78

if native support is not available, `options.nativeClientAvailable && options.nativeClientAvailable === false` will always be false because `options.nativeClientAvailable` is false, the transport is not set.

You may set a breakpoint in the `_transport` method in `raven-plugin.js`, and may notice it will never be reached.
It uses the original method of `raven.js`. In more recent version of react-native the method will work, but we are using react-native 0.42 and `withCredentials` of xhr is not added yet, the request will never be sent.

**Comment by HazAT** [comment]
Hey, this should be fixed in `0.17.0`
